pkg/virt-controller/watch/migration/migration.go

```diff
@@ -333,14 +333,16 @@ func (c *Controller) execute(key string) error {
 	}
 
 	if !vmiExists {
-		var err error
-
+		if migration.IsInterrupted() {
+			logger.V(3).Infof("Migration for deleted VMI %s/%s didn't finish. Keeping VMIM around so VMI controller can resolve issues on next start.", migration.Namespace, migration.Spec.VMIName)
+			return nil
```

Comment:
when will the VMIM object been cleaned-up if the vmi doesn't exist?

Reply:
The VMIM object will be cleanup up after recovery next time the VM is started.

Reply:
That's not correct. For the migrations created by the workload updater, there could be multiple vmims.

Reply:
Oh, I stand corrected, I'll add a check to get the active migration only